relay-redis/Cargo.toml

```diff
@@ -17,8 +17,8 @@ thiserror = "1.0.20"
 
 [features]
 default = []
-tls = ["redis/tls"]
-impl = ["r2d2", "redis"]
+tls = ["redis?/tls"]
```

Comment:
This doesn't need a `dep:`? I wasn't sure about this from reading the docs.

Reply:
yep, it's just requires `?` for the optional feature. 
it also gives nice error 
```
feature `tls` includes `dep:redis?/tls` with both `dep:` and `/`
  To fix this, remove the `dep:` prefix.
```